patchwork/urls.py

```diff
@@ -32,10 +32,15 @@
     path('admin/', admin.site.urls),
     path('', project_views.project_list, name='project-list'),
     path(
-        'project/<project_id>/list/',
+        'project/<project_id>/patches/',
         patch_views.patch_list,
         name='patch-list',
     ),
```

Comment:
If you rename this you need to provide a redirect. There are existing examples of redirects here. Don't forget to preserve query string arguments though since those matter here but didn't matter for fetching e.g. patch mboxes.

This should also be a separate commit, ideally placed before this one in the series.